data/index.html

```diff
@@ -5,133 +5,125 @@
+  <!-- CSS:Framework:Theme United -->
+  <link rel="stylesheet" href="bootstrap.min.css">
+  <!-- CSS:LoadingBar -->
+  <link rel="stylesheet" href="explorer.css">
 
 </head>
 
 <body>
 
-<a id="github-banner"
-   href="https://github.com/georgf/fx-data-explorer">
-   <img src="https://camo.githubusercontent.com/a6677b08c955af8400f44c6298f40e7d19cc5b2d/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f677261795f3664366436642e706e67" alt="Fork me on GitHub" data-canonical-src="https://s3.amazonaws.com/github/ribbons/forkme_right_gray_6d6d6d.png" />
-</a>
-
-<nav class="navbar navbar-expand-md navbar-inverse bg-inverse mb-4">
-  <a class="navbar-brand" href="#">Probe data</a>
-  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation">
-    <span class="navbar-toggler-icon"></span>
-  </button>
-  <div class="collapse navbar-collapse" id="navbarCollapse">
-    <ul class="navbar-nav mr-auto">
-      <li class="nav-item active">
-        <a class="nav-link" href="#probes">Find probes <span class="sr-only">(current)</span></a>
-      </li>
-      <li class="nav-item">
-        <a class="nav-link" href="#stats">Stats</a>
-      </li>
-    </ul>
+  <!-- GitHub:Fork -->
+  <!-- <a id="github-banner"
+     href="https://github.com/georgf/fx-data-explorer">
+     <img src="https://camo.githubusercontent.com/a6677b08c955af8400f44c6298f40e7d19cc5b2d/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f677261795f3664366436642e706e67" alt="Fork me on GitHub" data-canonical-src="https://s3.amazonaws.com/github/ribbons/forkme_right_gray_6d6d6d.png" />
+  </a> -->
+
+  <!-- Navbar -->
+  <div class="container-full">
+    <nav class="navbar navbar-toggleable-md navbar-inverse bg-primary">
+      <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <a class="navbar-brand" href="#">Probe data</a>
+      <div class="collapse navbar-collapse" id="navbarCollapse">
+        <ul class="navbar-nav">
+          <li class="nav-item active">
+            <a class="nav-link" href="index.html">Find probes <span class="sr-only">(current)</span></a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="stats.html">Stats</a>
+          </li>
+        </ul>
+      </div>
+    </nav>
   </div>
-</nav>
-
-<div id="overlay">
-  <img id="loading-anim" src="loading.gif">
-</div>
 
-<ul class="tabrow">
-  <li class="selected">find probes</li>
-  <li><a href="stats.html">stats</a></li>
-</ul>
-
-<div id="content">
+  <!-- Loading GIF, replace with bootstrap loadbar? -->
```

Comment:
Interesting, lets file a new follow-up issue for that?

Reply:
Yes, let's do that. I think that's something we can address as part of the new front-rend implementation using React+Redux.

Reply:
I opened #6 for that.